cypress/integration/steps.spec.js

```diff
@@ -87,9 +87,7 @@ context('Steps', () => {
     context('click log line in last step', () => {
       beforeEach(() => {
         cy.get('@stepHeaders').last().click({ force: true });
-        cy.get('[data-test=logs-5]').within(() => {
-          cy.get('[data-test=log-line-num-2]').as('lineNumber');
-        });
+        cy.get('[data-test=log-line-num-5-2]').as('lineNumber');
```

Comment:
adding the line number allows us to test specific lines easier, let cypress set up tests quicker